esx_updatedvehshop#1.sql

```diff
@@ -1,10 +1,10 @@
 USE `es_extended`;
 
-DELETE FROM 'vehicle_categories' WHERE name IN ('sedans', 'compacts');
+DELETE FROM `vehicle_categories` WHERE name IN ('sedans', 'compacts');
 
-DELETE FROM 'vehicles' WHERE model IN ('rumpo', 'stromberg', 'deluxo', 'ardent', 'oppressor', 'blazer5', 'ruiner2', 'voltic', 'monster');
+DELETE FROM `vehicles` WHERE model IN ('rumpo', 'stromberg', 'deluxo', 'ardent', 'oppressor', 'blazer5', 'ruiner2', 'voltic', 'monster');
 
-UPDATE 'vehicles'
+UPDATE `vehicles`
```

Comment:
This is a invalid SQL statement.

See: https://stackoverflow.com/questions/3667439/setting-multiple-column-using-one-update